README.md

```diff
@@ -29,13 +29,15 @@ Before you can start working with Yari, you need to:
 1.  Clone the forked repositories to your computer using the following commands
     (replace `[your account]` with the account you forked the repositories to):
 
-            git clone https://github.com/[your_account]/content.git
-            git clone https://github.com/[your_account]/yari.git
+```shell
+git clone https://github.com/[your_account]/content.git
+git clone https://github.com/[your_account]/yari.git
+```
 
-        Take a note of the file path to the location where you've cloned that
-        repo before moving on.
+Take a note of the file path to the location where you've cloned that
```

Comment:
I think this was expected to be part of the list item, and so should be indented (but by 4 spaces, not by 8)

Reply:
and the triple-backtick codeblock should also have this 4 space indentation (unlike in your "keep whitespace" attempt)

Reply:
Hmmm, the problem is if we indent it, it becomes a code snippet. 🤔 I'm not really sure how to indent it without this.

Reply:
&emsp;You can use \&emsp; to add space approximately equal to four spaces, as in this line.
&nbsp;&nbsp;&nbsp;&nbsp;You can alternatively use \&nbsp; four times in a row, as in this line.

Reply:
Wow, thanks a lot, @danielhjacobs !

Reply:
```suggestion
 Take a note of the file path to the location where you've cloned that
```